app/plugin_api.go

```diff
@@ -311,6 +311,21 @@ func (api *PluginAPI) UpdateUserStatus(userId, status string) (*model.Status, *m
 	return api.app.GetStatus(userId)
 }
 
+func (api *PluginAPI) UpdateUserStatusV2(userId, status, endtime string) (*model.Status, *model.AppError) {
```

Comment:
V2 means nothing, I would use something like `UpdateUserStatusWithDndTimeout` is a bit long, but at least describe why is there.